scene/gui/color_picker.h

```diff
@@ -119,14 +138,16 @@ class ColorPicker : public BoxContainer {
 
 	void _uv_input(const Ref<InputEvent> &p_event, Control *c);
 	void _w_input(const Ref<InputEvent> &p_event);
-	void _preset_input(const Ref<InputEvent> &p_event);
+	void _preset_input(const Ref<InputEvent> &p_event, Color p_button);
```

Comment:
`Color p_button` -> `const Color& p_color` or `p_button_color`

Reply:
Oops, thought I changed this. Fixed.